README.md

```diff
@@ -5,23 +5,27 @@ Visit http://webpy.org/ for more information.
 [![build status](https://secure.travis-ci.org/webpy/webpy.png?branch=master)](https://travis-ci.org/webpy/webpy)
 [![Codecov Test Coverage](https://codecov.io/gh/webpy/webpy/branch/master/graphs/badge.svg?style=flat)](https://codecov.io/gh/webpy/webpy)
 
-The latest stable release `0.51` supports both Python 2.7 and Python >= 3.5.
+The latest stable release `0.60` only supports supports Python >= 3.5.
```

Comment:
Duplicate “supports”?

Reply:
Thanks for pointing that. I'll correct it.

@iredmail Do you mind, if I merge this and fix it in the subsequent commit?

Reply:
I want to tag this commit and 0.60, as I've already published the release.

Reply:
Sure no problem.

Reply:
could you please aprove this PR?